gst/parser/gstttmlparse.c

```diff
@@ -1853,180 +1854,3 @@ gst_ttml_parse_change_state (GstElement * element, GstStateChange transition)
 
   return ret;
 }
-
-/*
- * Typefind support.
- */
```

Comment:
Not sure about this part, typefind does not appear to be part of the original TTML elements. Either way, this is at the element level, not plugin, so the majority of the code should still live here (if we keep it), possibly by just making `gst_ttmlparse_type_find()` globally or locally visible (instead of statically visible, hidden) and making the registration as before in the shared `plugin_init()`.